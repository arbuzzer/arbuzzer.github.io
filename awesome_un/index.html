<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Awesome Uniersity</title>
	<link rel="stylesheet" href="css/style.css">
	<link href="https://fonts.googleapis.com/css?family=Quicksand:400,700" rel="stylesheet">
</head>
<body>
	<header>
		<div class="container">
			<div class="top_menu">
				<img src="img/logo.png" alt="" width="20%">
				<nav>
					<a href="#">Specialty</a>
					<a href="#">Studying</a>
					<div class="social">
						<a href="http://instagram.com"><img src="img/instagram.png" alt="Instagram" width="18" height="18"></a>
						<a href="http://facebook.com"><img src="img/facebook.png" alt="Facebook" width="14" height="18"></a>
						<a href="http://twitter.com"><img src="img/twitter.png" alt="Twitter" width="13" height="18"></a>
					</div>
					<div class="language">
						<a href="#" class="l_en">English</a> | <a href="#">Español</a>
					</div>
					<button>Ask question</button>
				</nav>
			</div>	
		</div>	
	</header>
	<section>
		<div class="container">
		<div id="square"></div>
		<div id="rectangle"></div>
			<div class="content">
				<img src="img/students.png" alt="Students" width="43%" class="students">
				<div class="content_text">
					<h1>New Year!</h1>
					<h2>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</h2>
					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
					<div class="white_block">
						<p>145<br><span class="number_text">days</span></p><hr>
						<p>22<br><span class="number_text">topics</span></p><hr>
						<p>15<br><span class="number_text">teachers</span></p><hr>
						<p>8<br><span class="number_text">exams</span></p>
					</div>
					<img src="img/logo.png" alt="" width="35%" class="content_logo">
					<div class="content_menu">
						<nav>
							<a href="#">Specialty</a>
							<a href="#">Studying</a>
							<div class="social">
								<a href="http://instagram.com"><img src="img/instagram.png" alt="Instagram" width="18" height="18"></a>
								<a href="http://facebook.com"><img src="img/facebook.png" alt="Facebook" width="14" height="18"></a>
								<a href="http://twitter.com"><img src="img/twitter.png" alt="Twitter" width="13" height="18"></a>
							</div>
							<div class="language">
								<a href="#">English | Español</a>
							</div>
						</nav>
					</div>	
				</div>
			</div>
		</div>
	</section>
</body>
</html>