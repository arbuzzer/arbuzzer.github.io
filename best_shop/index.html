<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>BEST SHOP</title>
	<link rel="stylesheet" type="text/css" href="css/style.css">
	<!--<link rel="stylesheet/less" type="text/css" href="css/style.less">
	<script src="less.min.js" type="text/javascript"></script>-->
	<link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:400,700" rel="stylesheet">
</head>
<body>
	<header>
		<div class="container_850">	
			<div class="header_content">
				<h1 class="logo"><span class="logo_red">best</span> shop</h1>
				<nav>
					<ul>
						<a href="#"><li>about</li></a>
						<a href="#"><li>product</li></a>
						<a href="#"><li>contacts</li></a>
					</ul>
				</nav>	
			</div>	
		</div>	
	</header>
	<section class="bg_top">
		<div class="container">
			<div class="bg">
				<h3>The Best clothes in our shop</h3>
				<h3>We have all you need</h3>
				<h3>Just for your and your family</h3>
				<button>go to shop</button>
			</div>
		</div>
	</section>
	<section class="bg_products">
		<div class="container">
			<div class="products">
				<div class="product">
					<div class="dress">
						<img src="img/dress1.jpg" alt="" class="dress1">
						<span class="info_icon">i</span>
						<div class="info">
							<p>Price: 300$</p>
							<p>Size: S,M</p>
							<p>Season: Autumn</p>
							<p>Material: Viscose</p>
						</div>
					</div>
					<div class="product_text">
						<h3>Blue dress</h3>
						<p>
							Awesome blue dress for strong but sweet woman. For you and you, and you and you friends and for that ugly girl
						</p>
						<button>View more</button>
					</div>	
				</div>
				<div class="product">
					<div class="dress">
						<img src="img/dress2.jpg" alt="">
						<div class="info">&#9432;</div>
					</div>
					<div class="product_text">
						<h3>Red dress</h3>
						<p>
							Awesome red dress for strong but sweet woman. For you and you, and you and you friends and for that ugly girl
						</p>
						<button>View more</button>
					</div>	
				</div>
				<div class="product">
					<div class="dress">
						<img src="img/dress3.jpg" alt="">
						<div class="info">&#9432;</div>
					</div>
					<div class="product_text">
						<h3>Just dress</h3>
						<p>
							Awesome dress for strong but sweet woman. For you and you, and you and you friends and for that ugly girl
						</p>
						<button>View more</button>
					</div>	
				</div>
			</div>	
		</div>	
	</section>
	<footer>
		<div class="container_850">
			<div class="footer_content">
				<h1 class="logo"><span class="logo_red">best</span> shop</h1>
				<div class="socail">
					<a href="http://twitter.com"><img src="img/twitter-logo-button.png" alt="" width="20%" class="twitter"></a>
					<a href="http://instagram.com"><img src="img/instagram-logo.png" alt="" width="20%"></a>
				</div>
			</div>	
		</div>
	</footer>
</body>
</html>