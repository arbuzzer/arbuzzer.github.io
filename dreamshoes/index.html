<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Dream Shoes</title>
	<link rel="stylesheet" href="css/style.css">
	<link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
</head>
<body>
	<header>
		<p class="b_logo">Не видно картинок? <a href="#" class="under">Нажмите здесь</a></p>
		<p class="logo_site"><img src="img/logo_site.png" alt="DreamShoes Ukraine"></p>
		<nav>
			<a href="#">кроссовки</a>
			<a href="#">кроссовки на осень</a>
			<a href="#">ugg australia</a>
			<a href="#">условия доставки</a>
		</nav>
		<div class="customer">
			<h3>Добрый день, Анна!</h3><br>
			<p>Минус <span class="red_text">150 гривен</span> на весь ассортимент!<br>Вводи промо-код: <span class="black_block">dream77</span> и активируй скидку.</p><br>
			<p>Не откладывай!<br>Условия действительны только <span class="red_text">до 15.10</span>.</p>
		</div>
		<div class="hr"></div>
	</header>	
	<section class="product">
		<h1>Одна модель для него и для нее</h1>
		<div class="size">
			<button type="submit" class="size_number">36</button>
			<button type="submit" class="size_number">37</button>
			<button type="submit" class="size_number">38</button>
			<button type="submit" class="size_number">39</button>
			<button type="submit" class="size_number">40</button>
			<button type="submit" class="size_number">41</button>
			<button type="submit" class="size_number">42</button>
			<button type="submit" class="size_number">43</button>
			<button type="submit" class="size_number">44</button>
			<button type="submit" class="size_number">45</button>
		</div>
		<div class="models">
			<div class="models_shoes adidas">
				<img src="img/img1.png" alt="Adidas">
				<p>Классика с черными полосками.<br><b>36-44 размеры</b></p>
				<div class="red_line"></div>
			</div>
			<div class="models_shoes nike">
				<img src="img/img2.png" alt="Nike Air Max 97 Ultra 17">
				<p>Классика Nike Air Max 97 Ultra 17<br><b>36-45 размеры</b></p>
				<div class="red_line"></div>
			</div>
		</div>
	</section>	
	<section class="fila_product">
		<div class="fila_text">
			<h1>
				Корейское качество и стиль
				от итальянских мастеров
			</h1><br>
			<p>
				это кроссовки Fila Disruptor Low White
			</p>
		</div>
		<div class="fila_img">
			<p class="fila_shoes"><img src="img/img3.png" alt="Fila Disruptor Low White"></p>
			<div class="fila_logo">
				<img src="img/logo_fila.png" alt="Fila Logo">
			</div>	
		</div>
	</section>
	<section class="vans_product">
		<div class="vans_shoes">
			<img src="img/img4.png" alt="Vans">
		</div>	
		<div class="vans_text">
			<p>вечная классика</p>
			<div class="vans_line"></div>
			<h1>vans</h1>
			<h5>off the wall</h5>
			<button class="vans_button" type="submit">в каталог</button>
		</div>
		<div class="vans_offer">
			<div class="vans_offer_text">
				<h3>
					Vans Old Skool<br>мужские
				</h3>
				<div class="red_text"></div>
				<p>
					Первая модель кед Old Skool<br>с узнаваемой полоской Vans.<br>Вечная классика, актуальная<br>при любых веяниях моды!
				</p>
			</div>
			<div class="vans_offer_img">
				<img src="img/img5.png" alt="Vans Old Skool">
			</div>
		</div>
	</section>
	<section class="nike">
		<div class="nike_offer">
			<div class="nike_offer_img">
				<img src="img/img6.png" alt="Nike Air Max 1/97">
			</div>
			<div class="nike_offer_text">
				<h3>
					Мужские кроссовки<br>Nike Air Max 1/97
				</h3>
				<div class="red_text"></div>
				<p>
					Победитель народного<br>голосования Vote Forward 2018,<br>с лучшим винтажным дизайном!
				</p>
			</div>
		</div>	
		<div class="nike_270">
			<div class="nike_air_text">
				<h3>
					Кроссовки Nike Air Max 270 мужские
				</h3>
				<div class="red_text"></div>
				<p>
					Свобода движения от Air Max 270! Воздушная подушка в 32 мм обеспечивает<br>супер комфортную ходьбу. Дизайн кроссовок – это сочетание двух легендарных<br>моделей 90-х годов Air Max 180 и Air Max 93.
				</p>
			</div>
			<div class="nike_270_img">
				<div class="nike_270_img_column">
					<img src="img/img7.png" alt="Nike Air Max 270 Purple">
					<img src="img/img8.png" alt="Nike Air Max 270 Green">
					<img src="img/img9.png" alt="Nike Air Max 270 Grey">
				</div>
				<img src="img/img10.png" alt="Nike Air Max 270 Black">
			</div>
		</div>
		<div class="nike_air">
			<div class="nike_air_text">
				<h3>
					Nike Air Max 97 Ultra
				</h3>
				<div class="red_text"></div>
				<p>
					Культовая модель 1997-го года с самой большой воздушной подушкой Air.<br>Получите максимум комфорта! Дизайн подчеркнут стремительными линиями,<br>которые вдохновлены высокоскоростным японским «поездом-пулей» Синкансэн.
				</p>
			</div>
			<div class="nike_97_img">
				<div class="nike_97 nike_color_woman">
					<img src="img/img11.png" alt="Nike Air Max 97 Ultra Red">
					<p>Женские Air Max<br>97 от 1485 грн</p>
					<button class="nike_button" type="submit">в каталог</button>
				</div>
				<div class="nike_97 nike_color_man">
					<img src="img/img12.png" alt="Nike Air Max 97 Ultra Blue">
					<p>Мужские Air Max<br>97 от 1480 грн</p>
					<button class="nike_button" type="submit">в каталог</button>
				</div>
			</div>
		</div>
	</section>	
	<section class="advantages">
		<div class="adv_content">
			<img src="img/wallet.png" alt="Wallet">
			<p>Оплата при получении</p>
		</div>
		<div class="adv_content">
			<img src="img/like.png" alt="Like">
			<p>Возврат<br> 365 дней</p>
		</div>
		<div class="adv_content">
			<img src="img/meter.png" alt="Meter">
			<p>Доставка<br> за сутки</p>
		</div>
		<div class="adv_content">
			<img src="img/flag.png" alt="Flag">
			<p>Гарантия на весь товар</p>
		</div>
	</section>
	<section class="company">
		<div class="company_row first_row">
			<div class="company_block">
				<img src="img/vans.png" alt="Vans">
			</div>
			<div class="company_block">
				<img src="img/timberland.png" alt="Timberland">
			</div>
			<div class="company_block">
				<img src="img/reebok.png" alt="Reebok">
			</div>
			<div class="company_block">
				<img src="img/ugg.png" alt="UGG">
			</div>
		</div>	
		<div class="company_row">
			<div class="company_block">
				<img src="img/converse.png" alt="Converse">
			</div>
			<div class="company_block">
				<img src="img/nike.png" alt="Nike">
			</div>
			<div class="company_block">
				<img src="img/asics.png" alt="Asics">
			</div>
			<div class="company_block">
				<img src="img/puma.png" alt="Puma">
			</div>
		</div>	
	</section>
	<footer>
		<div class="contact">
			<div class="black">
				<p><b>График работы:</b><br>Заказы online 24/7<br>10:00-19:00 без выходных</p>
			</div>
			<div class="grey">
				<div class="phone_number">
					<p><img src="img/phone.png" alt="Phone"><p>
					<p><b>093</b> 170 13 21<br><b>099</b> 606 38 86<br><b>067</b> 343 66 16</p>
				</div>	
			</div>
			<button type="submit" class="red">
				<p><img src="img/cart.png" alt="Cart"></p>
				<p><b>Мой заказ</b></p>
			</button>
		</div>
		<div class="footer_text">
			<p>
				Вы получили это письмо, потому что подписаны на рассылку <a href="#" class="red_text">dreamshoes.in.ua</a>.<br>Если Вы не хотите получать рассылку, перейдите по ссылке: <a href="#" class="under">отписаться</a>.
			</p><br>
			<p>
				© 2018 Интернет-магазин cпортивной обуви Дримшуз
			</p>
		</div>
	</footer>
</html>