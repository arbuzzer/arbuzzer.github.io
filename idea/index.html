<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>IDEA</title>
	<link rel="stylesheet" href="css/style.css">
	<link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,700" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Montserrat:700" rel="stylesheet">
</head>
<body>
	<header>
		<div class="container">
			<img src="img/logo.png" alt="Logo" class="logo">
			<div class="title">
				<h1>Creative Presentation Ideas</h1>
				<p>We take pride in delivering Intelligent Designs and Engaging Experiences for clients all over the World.</p>
				<input type="button" value="Learn more">
			</div>
		</div>
	</header>
	<section id="grey">
		<div class="container">
			<img src="img/logo.png" alt="Logo" class="logo">
			<div class="first">
				<div class="first_text">
					<h1>Design & Direction</h1>
					<p>Graphic Designer with a focus on interactive design, helping clients communicate and solve problems through creative visual solutions.</p>
					<input type="button" value="See Portfolio">
				</div>
			</div>
		</div>
	</section>
	<section id="second">
		<div class="container">
			<img src="img/logo.png" alt="Logo" class="logo">
			<div class="second">
				<div class="second_content">
					<div class="second_text">
						<h1>
							Usefull & Semplicity
						</h1>
						<p>
							Graphic Designer with a focus on interactive design, helping clients communicate and solve problems through creative visual solutions.
						</p>
					</div>
					<div class="second__item">
						<div class="second_item">
							<img src="img/switch.png" alt="">
							<p>Row Setting</p>
						</div>
						<div class="second_item">
							<img src="img/settings.png" alt="">
							<p>Major Setting</p>
						</div>
						<div class="second_item">
							<img src="img/add_app.png" alt="">
							<p>Nice Themes</p>
						</div>
					</div>
				</div>
				<img src="img/iphone.png" alt="iPhone" class="iphone">
			</div>
		</div>
	</section>
	<section id="blue">
		<div class="container">
			<img src="img/logo.png" alt="Logo" class="logo">
			<div class="blue">
				<div class="blue_text">
					<h1>We craft a unique identity that best conveys your values.</h1>
					<div class="blue_text_p">
						<p>Massive Dynamic has over 10 years of experience in Design, Technology and Marketing.
						We create successful, innovative digital products from the initial concept through full product launch.</p>
						<p>We take pride in delivering Intelligent Designs and Engaging Experiences for clients all over the World.</p>
					</div>
				</div>
				<div class="blue_content">
					<div class="blue_item">
						<img src="img/vector1.png" alt="">
						<p>Beautiful Design</p>
					</div>
					<div class="blue_item">
						<img src="img/vector2.png" alt="">
						<p>Corporate Identity</p>
					</div>
					<div class="blue_item">
						<img src="img/vector3.png" alt="">
						<p>Beautiful Design</p>
					</div>
					<div class="blue_item">
						<img src="img/vector4.png" alt="">
						<p>Mobile Freindly</p>
					</div>
					<div class="blue_item">
						<img src="img/vector5.png" alt="">
						<p>Beautiful Design</p>
					</div>
					<div class="blue_item">
						<img src="img/vector3.png" alt="">
						<p>Beautiful Design</p>
					</div>
				</div>
			</div>
		</div>
	</section>
	<footer>
		<div class="container">
			<img src="img/logo.png" alt="Logo" class="logo">
			<div class="footer_info">
				<div class="footer_left">
					<div class="footer_text">
						<h1>
							Get in Touch
						</h1>
						<p>
							We are always looking for new exciting projects and collaborations. Feel free to contact us.
						</p>
					</div>
					<div class="footer_contact">
						<div class="office">
							<h1>
								Office
							</h1>
							<p>
								2901 Marmora road, Glassgow,<br>
								Seattle, WA 98122-1090
							</p>
						</div>
						<div class="contact">
							<h1>
								Contact
							</h1>
							<p>
								+01  34 44 5880
								massive@example.com
							</p>
						</div>
						<div class="footer_item footer_item_adapt">
							<p>Follow us</p>
							<a href="https://twitter.com/">
								<img src="img/twitter.png" alt="">
							</a>
							<a href="https://www.instagram.com/">
								<img src="img/instagram.png" alt="">
							</a>
							<a href="https://www.linkedin.com/">
								<img src="img/linkedin.png" alt="">
							</a>
							<a href="https://dribbble.com/">
								<img src="img/dribbble.png" alt="">
							</a>
						</div>
					</div>
					<div class="footer_item footer_item_desk">
							<p>Follow us</p>
							<a href="https://twitter.com/">
								<img src="img/twitter.png" alt="">
							</a>
							<a href="https://www.instagram.com/">
								<img src="img/instagram.png" alt="">
							</a>
							<a href="https://www.linkedin.com/">
								<img src="img/linkedin.png" alt="">
							</a>
							<a href="https://dribbble.com/">
								<img src="img/dribbble.png" alt="">
							</a>
						</div>
				</div>
				<div class="footer_right">
					<div class="top_input">
						<input type="text" placeholder="Send">
						<input type="text" placeholder="Email">
						<input type="text" placeholder="Subject">
					</div>
					<input type="text" placeholder="Message" class="mes_in">
					<button class="mes_but">Message</button>
				</div>
			</div>	
		</div>
	</footer>
</body>
</html>