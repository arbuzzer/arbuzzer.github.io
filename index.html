<!DOCTYPE html>
<html>
<head>
	<meta charset="UTF-8">
	<title>Интернет-магазин LEO SHOP</title>
	<link rel="stylesheet" href="libs/reset.css">
	<link rel="stylesheet" href="libs/960_12_col.css">
	<link rel="stylesheet" href="css/style.css">
	<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
	<link rel="stylesheet" href="css/icomoon.css">
</head>
<body>
	<header>
		<div class="container_12">
			<div class="grid_5">
				<ul class="top_menu">
					<li class="lang">
					Language: <a href="#">English <span class="icon-down-arrow"></span></a>
					</li>
					<li class="currency">
					Currency: <a href="#">USD <span class="icon-down-arrow"></a>
					</li>
			</ul>
			</div>
			<div class="grid_7">
				<nav class="menu">
					<ul>
						<li><a href="#">Account</a></li>
						<li><a href="#">Wishlist</a></li>
						<li><a href="#">Checkout</a></li>
						<li><a href="#">Log In</a></li>
						<li><a href="#">Sign Up</a></li>
					</ul>
				</nav>
			</div>
		</div>
		<div class="container_12">
			<div class="grid_2">
				<a href="#" class="logo"><img src="img/logo.png" alt="Интернет-магазин LEO SHOP"></a>
			</div>
			<div class="grid_6">
				<nav class="mnu_top">
					<ul>
						<li>
							<a href="#">HOME</a>
						</li>
						<li>
							<a href="#">WOMEN</a>
						</li>
						<li>
							<a href="#">MEN</a>
						</li>
						<li>
							<a href="#">OTHER</a>
						</li>
						<li>
							<a href="#">PURCHASE</a>
						</li>
					</ul>
				</nav>
			</div>
			<div class="grid_4">
				<div class="search">
					<form action="#">
						<input type="text" class="form-control search_input" placeholder="Search..." />
						<input type="submit" value="&#xe986;" class="search_button" />
					</form>
				</div>
				<a href="#">
					<img class="top_1" src="img/top_1.png" alt="Заказы">
				</a>
				<a href="#">
					<img class="top_2" src="img/top_2.png" alt="Корзина">
				</a>
			</div>
		</div>
	</header>
	<section class="top_banner">
		<div class="container_12">
			<div class="grid_2">
				<div class="left">
					<img src="img/left.png" alt="alt">
				</div>
			</div>
			<div class="grid_4">
				<h2 class="banner_text">
				ALUMINUM CLUB <br><b>Experience Ray-Ban</b>
			</div>
			<div class="grid_6">
				<div class="right">
					<img src="img/right.png" alt="alt">
				</div>
			</div>
		</div>
	</section>
	<section>
		<div class="container_12">
			<div class="grid_4">
				<div class="featured">
				FEATURED PRODUCTS
				</div>
			</div>
		</div>
	</section>
	<section>
		<div class="container_12">
			<div class="grid_12">
				<div class="container-fluid">
					<div class="signup">
						<img src="img/sign_up.png" alt="">
						<h1>
							<span>NEWSLETTERS<br>SIGNUP</span>
						</h1>
						<p>
							<span class="span">Lorem ipsum dolor sit amen,<br>dolor imun ra adlip  ulissys </span>
						</p>
						<form action="URL">
							<input type="text">
							<a href="#">Subscribe</a>
						</form>
					</div>
				</div>
			</div>
		</div>
	</section>
	<footer>
		
	</footer>
	<script src="libs/jquery-3.3.1.min.js" type="text/javascript"></script>
	<script src="libs/query.jcarousel.min.js" type="text/javascript"></script>
	<script src="js/common.js" type="text/javascript"></script>
	<script src="libs/html5shiv.min.js" type="text/javascript"></script>
</body>
</html>