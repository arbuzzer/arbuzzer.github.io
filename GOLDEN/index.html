<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Golden Studio</title>
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
		<header>
			<div class="container">
				<div class="heading">
					<a href="#">
						<img src="img/logo.png" alt="Golden" class="logo">
					</a>
					<nav id="menu" class="menu">
						<div class="topnav">
							<a href="index.html">Home</a>
							<a href="#services">Services</a>
							<a href="#portfolio">Portfolio</a>
							<a href="#about">About</a>
							<a href="#contact">Contact</a>
						</div>
					</nav>
				</div>
				<div class="titles">
					<div class="title_first">
						Welcome to our studio!
					</div>
					<h1>
						It's nice to Meet you
					</h1>
				</div>
				<a class="button" href="#">Tell Me More</a>
			</div>
		</header>
		<section id="services">
			<div class="container">
				<div class="title">
					<h2>
						Services
					</h2>
					<p>
						Proin iaculispurus consequat sem cure.
					</p>
				</div>
				<div class="services">
					<div class="services_item">
						<img src="img/icon1.png" alt="Услуга">
						<h3>E-Commerce</h3>
						<p>
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt.
						</p>
					</div>
					<div class="services_item">
						<img src="img/icon2.png" alt="Услуга">
						<h3>Responsive Web</h3>
						<p>
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt.
						</p>
					</div>
					<div class="services_item"
					>
						<img src="img/icon3.png" alt="Услуга">
						<h3>Web Security</h3>
						<p>
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt.
						</p>
					</div>
				</div>
			</div>
		</section>
		<section id="portfolio">
			<div class="container">
				<div class="title">
					<h2>
						Our Portfolio
					</h2>
					<p>
						Proin iaculis purus consequat sem cure. 
					</p>
				</div>
				<div class="works">
					<img src="img/img1.jpg" alt="Работа">
					<img src="img/img2.jpg" alt="Работа">
					<img src="img/img3.jpg" alt="Работа">
				</div>
			</div>
		</section>
		<section id="about">
			<div class="container">
				<div class="title">
					<h2>
						About Us
					</h2>
					<p>
						Proin iaculis purus consequat sem cure. 
					</p>
				</div>
				<div class="about">
					<div class="about_item1 about_item">
						<img src="img/about1.jpg" alt="Услуга" class="img_about_adapt">
						<div class="about_item_text1 about_item_text">
							<h3>January 2011<br>Facing Startup Battles</h3>
							<p>
								Proin iaculis purus consequat sem cure<br>
								digni ssim. Donec porttitora entum suscipit<br>
								aenean rhoncus posuere odio in tincidunt. Proin<br> 
								iaculis purus consequat sem cure digni<br>
								ssim. Donec porttitora entum suscipit.
							</p>
						</div>
						<img src="img/about2.jpg" alt="Услуга" class="img_about_table">
					</div>
					<div class="about_item2 about_item">
						<img src="img/about3.jpg" alt="Услуга">
						<div class="about_item_text2 about_item_text">
							<h3>December 2012<br>Enter The Dark Days</h3>
							<p>
								Proin iaculis purus consequat sem cure<br>
								digni ssim. Donec porttitora entum suscipit<br>
								aenean rhoncus posuere odio in tincidunt. Proin<br>
								iaculis purus consequat sem cure digni<br>
								ssim. Donec porttitora entum suscipit aenean<br>
								rhoncus posuere odio in tincidunt.
							</p>
						</div>
					</div>
					<div class="about_item3 about_item">
						<img src="img/about4.jpg" alt="Услуга" class="img_about_adapt">
						<div class="about_item_text3 about_item_text">
							<h3>February 2014<br>Our Triumph</h3>
							<p>
								Proin iaculis purus consequat sem cure<br> 
								digni ssim. Donec porttitora entum suscipit<br>
								aenean rhoncus posuere odio in tincidunt. Proin<br> 
								iaculis purus consequat sem cure digni<br>
								ssim. Donec porttitora entum suscipit aenean.
							</p>
						</div>
						<img src="img/about4.jpg" alt="Услуга" class="img_about_table">
					</div>
					<div class="about_item4 about_item">
						<img src="img/about1.jpg" alt="Услуга">
						<div class="about_item_text2 about_item_text">
							<h3>December 2012<br>We are Leaders!</h3>
							<p>
								Proin iaculis purus consequat sem cure<br>
								digni ssim. Donec porttitora entum suscipit<br>
								aenean rhoncus posuere odio in tincidunt. Proin<br>
								iaculis purus consequat sem cure digni<br>
								ssim. Donec porttitora entum suscipit aenean<br>
								rhoncus posuere odio in tincidunt.
							</p>
						</div>
					</div>
				</div>
			</div>
		</section>
		<footer id="contact">
			<div class="copyright">
				<p>
					© Copyright 2014 FreebiesXpress.com
				</p>
			</div>
		</footer>
		<script src="js/common.js"></script>
</body>
</html>